hail/python/hail/backend/service_backend.py

```diff
@@ -157,6 +157,8 @@ async def serialize(self, writer: afs.WritableStream):
 
 
 class ServiceBackend(Backend):
+    input_version = 2
```

Comment:
How come this versioning here? Seems arbitrary, is it ever checked? Are there instructions on how or when it should be updated?

Reply:
Sorry. It was old cruft from when I thought I had to make the spec backwards compatible.